src/App.vue

```diff
@@ -1,32 +1,54 @@
+
 <template>
-  <div id="app">
-    <ul>
-      <li><router-link to="/">Main</router-link></li>
-      <li><router-link to="/About">About</router-link></li>
-    </ul>
-    <router-view></router-view>
-    <div class="footer">
-      <h2>This appears after router-view</h2>
+    <div id="app" class="container-fluid">
+    
+        <nav class="navbar fixed-top navbar-light bg-faded customNav">
+            <div class="navbar-header">
+                <div class="row">
+                    <div class="col-md-4 hidden-sm-down">
```

Comment:
Why the repeated columns and hiding/showing? Is it just for the center alignment?